### PR TITLE
Add integration tests

### PR DESCRIPTION
**Pull request by Xodoc**

### CODE REVIEW

File: ConsoleApp/IntegrationTests/Services/WeatherServiceIntegrationTest.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            _weatherService = new WeatherService(_weatherRepository, _validator);
+        }
+
+        [Theory]
+        [InlineData("Yakutsk")]
+        [InlineData("Morocco")]
+        public async void GetWeatherAsync_WhenSendingCorrectCityName_GettingWeatherMessage(string cityName) 
+        {
+            //Arrange
+            var weather = await _weatherRepository.GetWeatherAsync(cityName);
```
Comment (denja2) [inline ConsoleApp/IntegrationTests/Services/WeatherServiceIntegrationTest.cs:36]: remove

File: ConsoleApp/IntegrationTests/Services/WeatherServiceIntegrationTest.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            var expectedMessage = $"{weather.main.temp}°C now.";
```
Comment (denja2) [inline ConsoleApp/IntegrationTests/Services/WeatherServiceIntegrationTest.cs:37]: extend the message

File: ConsoleApp/IntegrationTests/Services/WeatherServiceIntegrationTest.cs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+            //Act
+            var result = await _weatherService.GetWeatherAsync(cityName);
+            var actualResult = Regex.Match(result, @"([0-9])(.*?)\.|(\B\W)(.*?)\.", RegexOptions.IgnorePatternWhitespace).Value;
```
Comment (denja2) [inline ConsoleApp/IntegrationTests/Services/WeatherServiceIntegrationTest.cs:41]: arrange and fix the regex

File: ConsoleApp/IntegrationTests/Services/WeatherServiceIntegrationTest.cs (hunk 1 above)
Comment (Xodoc) [reply ConsoleApp/IntegrationTests/Services/WeatherServiceIntegrationTest.cs:36]: done

File: ConsoleApp/IntegrationTests/Services/WeatherServiceIntegrationTest.cs (hunk 2 above)
Comment (Xodoc) [reply ConsoleApp/IntegrationTests/Services/WeatherServiceIntegrationTest.cs:37]: done

File: ConsoleApp/IntegrationTests/Services/WeatherServiceIntegrationTest.cs (hunk 3 above)
Comment (Xodoc) [reply ConsoleApp/IntegrationTests/Services/WeatherServiceIntegrationTest.cs:41]: done

File: ConsoleApp/IntegrationTests/Services/WeatherServiceIntegrationTest.cs [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            _weatherService = new WeatherService(_weatherRepository, _validator);
+        }
+
+        [Theory]
+        [InlineData("Yakutsk")]
+        [InlineData("Morocco")]
+        public async void GetWeatherAsync_WhenSendingCorrectCityName_GettingWeatherMessage(string cityName)
+        {
+            //Arrange
+            string[] des = new string[]
```
[the remark this answers is not in the document]
Comment (Xodoc) [reply ConsoleApp/IntegrationTests/Services/WeatherServiceIntegrationTest.cs:35]: done

**Pull request merged**